recipes/kubernetes-client-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: kubernetes
```

Comment:
We might want to change the package name to `kubernetes-python` to disambiguate the python client from other clients, although it looks like there is a `python-kubernetes` on pypi, so this might cause issues. @jakirkham -- thoughts?